Docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.9'
+services:
+  postgres:
+    image: 'postgres:14.2-alpine'
+    ports:
+      - '5432:5432'
+    environment:
```

Comment:
why not to use .env from file?

Reply:
By default `.env` file is used. With `--file-env` you can change the env file to another one.

> If you have multiple environment variables, you can substitute them by adding them to a default environment variable file named .env or by providing a path to your environment variables file using the --env-file command-line option.
[More here](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)

In any case, you need to provide variables from env using the `$` sign in the Docker-compose file.